tests/base/hdf5_03.cc

```diff
@@ -803,14 +803,6 @@ main(int argc, char **argv)
                                                       pcout);
 #endif
 
-        write_test<std::vector, int>(data_file,
-                                     dataset_dimensions,
-                                     mpi_communicator,
-                                     pcout);
-        write_test<std::vector, unsigned int>(data_file,
-                                              dataset_dimensions,
-                                              mpi_communicator,
-                                              pcout);
```

Comment:
Because I removed many instantiations, I had to modify the tests hdf5_03 and and hdf5_04